asttokens/util.py

```diff
@@ -59,21 +68,24 @@ def expect_token(token, tok_type, tok_str=None):
 
 # These were previously defined in tokenize.py and distinguishable by being greater than
 # token.N_TOKEN. As of python3.7, they are in token.py, and we check for them explicitly.
-if hasattr(token, 'ENCODING'):
+if sys.version_info >= (3, 7):
```

Comment:
Because of https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks